recipes/r-getpass/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = '0.2-2' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-getpass
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: getPass_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/getPass_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/getPass/getPass_{{ version }}.tar.gz
+  sha256: 8cfdc7e8cabeae5dd325015d735b139d9828bd54ea8e7c2dcb636a00a153cd0f
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rstudioapi >=0.5
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-rstudioapi >=0.5
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('getPass')"  # [not win]
+    - "\"%R%\" -e \"library('getPass')\""  # [win]
+
+about:
+  home: https
+  license: BSD 2-clause
+  summary: A micro-package for reading "passwords", i.e.  reading user input with masking, so
+    that the input is not displayed as it  is typed.  Currently we have support for
+    'RStudio', the command line (every OS), and any platform where 'tcltk' is present.
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
You can use the same strategy as GPL. File name is `BSD_2_clause`

Reply:
@isuruf is there a strict guideline for license_file? Afaik BSD does not require to ship the license, right?

Reply:
> You can use the same strategy as GPL. File name is BSD_2_clause

@isuruf The file shipped with r-base is a template. Is that sufficient?

https://cran.r-project.org/web/licenses/BSD_2_clause
https://cran.r-project.org/web/packages/getPass/LICENSE

> Afaik BSD does not require to ship the license, right?

@bgruening I included it because of this sentence: "Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution."

Reply:
@jdblischak, you are right. A template isn't sufficient

Reply:
Fair enough!